Solutions/DNS Essentials/NgrokReverseProxyOnNetwork

```diff
@@ -0,0 +1,30 @@
+id: 50b0dfb7-2c94-4eaf-a332-a5936d78c263
+name: Ngrok Reverse Proxy on Network (ASIM DNS Solution)
```

Comment:
I think we are missing few properties like queryFrequency, queryPeriod etc.
Please refer to this template for updating required properties